glide-core/telemetry/src/open_telemetry.rs

```diff
@@ -68,28 +69,82 @@ fn parse_endpoint(endpoint: &str) -> Result<GlideOpenTelemetryTraceExporter, Err
+        "grpc" => Ok(GlideOpenTelemetryTraceExporter::Grpc(endpoint.to_string())), // gRPC endpoint
+        "file" => {
+            // For file, we need to extract the path without the 'file://' prefix
+            let file_prefix = "file://";
+            if !endpoint.starts_with(file_prefix) {
+                return Err(Error::new(
+                    ErrorKind::InvalidInput,
+                    "File path must start with 'file://'",
+                ));
+            }
+
+            // Extract the path by removing the 'file://' prefix
+            let path = endpoint.strip_prefix(file_prefix).ok_or_else(|| {
+                Error::new(
+                    ErrorKind::InvalidInput,
+                    "Failed to extract path from file URL",
+                )
+            })?;
+
+            let path_buf = PathBuf::from(path);
+
+            // Determine if this is a directory path or a file path
+            let final_path = if path_buf.is_dir() || path_buf.extension().is_none() {
+                // If it's a directory or doesn't have an extension, treat it as a directory
+                // and append the default filename
+                path_buf.join("spans.json")
+            } else {
+                path_buf
+            };
+
+            // Check if the parent directory exists and is a directory
+            if let Some(parent_dir) = final_path.parent() {
+                match parent_dir.try_exists() {
+                    Ok(exists) => {
+                        if !exists {
+                            // Try to create the parent directory if it doesn't exist
+                            if let Err(e) = std::fs::create_dir_all(parent_dir) {
+                                return Err(Error::new(
+                                    ErrorKind::InvalidInput,
+                                    format!(
+                                        "Parent directory does not exist and could not be created: {} - {}",
+                                        parent_dir.display(), e
+                                    ),
+                                ));
+                            }
+                        } else if !parent_dir.is_dir() {
+                            return Err(Error::new(
+                                ErrorKind::InvalidInput,
+                                format!(
+                                    "Parent path exists but is not a directory: {}",
+                                    parent_dir.display()
+                                ),
+                            ));
+                        }
+                    }
+                    Err(e) => {
+                        return Err(Error::new(
+                            ErrorKind::InvalidInput,
+                            format!("Error checking if parent directory exists: {}", e),
+                        ));
+                    }
+                }
+            }
```

Comment:
We don't need to check if the folder exists here - we can simply try to create/open the file and if it fails return the error message. As we talked - we shouldn't create folders, only the file itself